docs/dev/zksync-v2/l1-l2-interop.md

```diff
@@ -44,19 +44,19 @@ Thirdly, the operator can not commit to processing each and every transaction wi
 
 In other words, we require the operator to do its best instead of requiring a strict deadline. The measure of "the work" is still to be developed. Most likely it will be the number of `ergs` the priority operations used.
 
-Fourthly, there needs to be a way to bypass spam attacks for important transactions. From the points above we already know that there is no strict deadline for processing the priority queue transactions. If the deque with priority gets filled with a lot of heavy transactions, the next transactions will need to wait until all these are processed. If you need to pass your transaction urgently, you can submit your transaction to a separate priority heap, where the transactions will be processed based on the layer 2 tip (`tx.value - txBaseCost * gasPrice`) provided to the operator. The transactions in the heap get processed before transactions in the queue.
+In the future, we will also add the ability to "prioritize" L1->L2 transactions, allowing users to speed the inclusion of their transaction in exchange for paying bigger fee to the operator.
 
 ### Summary
 
-For each priority queue transaction, besides supplying the transaction input data, the user also provides the following:
-
-- The queue type to use. It is either a `Deque`, `HeapBuffer` or `Heap`. `Deque` is used for sequential processing of the transactions, while `HeapBuffer` and `Heap` are used for fee-prioritized processing of transactions. For the testnet, only `Deque` is available. The details on using `HeapBuffer` and `Heap` will be available later with their release.
-- The operator fee for the transaction as part of the transaction `value` (the amount of ETH passed with the L1 transaction).
+- L1->L2 are not free for the users.
+- Unlike the counterpart from the previous version of zkSync, all kinds of transactions are enabled. 
```

Comment:
```suggestion
- Unlike the counterpart from the previous version of zkSync, any transaction that can be initiated from L2, can also be initiated from L1. 
```

Reply:
Hmm, this is not completely true. You can initiate account attraction transactions from L2, you can't do this on L1

Reply:
But then what does "all kinds of transactions" mean?

Reply:
Anyways I guess the phrasing should be made a bit less abstract.

Reply:
I'll probably just remove the summary. It made more sense when we had more content in the section about the priority queue, but now it does not seem to